src/tmpfiles/tmpfiles.c

```diff
@@ -3684,7 +3684,7 @@ static int parse_line(
                 else {
                         *invalid_config = true;
                         return log_syntax(NULL, LOG_ERR, fname, line, SYNTHETIC_ERRNO(EBADMSG),
-                                          "Unknown modifiers in command '%s'.", action);
+                                          "unknown modifiers in command: %s", action);
```

Comment:
why make the first letter lowercase? All other messages here are uppercase, as we usually do

Reply:
I think when have a prefix, we shouldn't capitalize the sentence.
"/usr/lib/tmpfiles.d/20-systemd-shell-extra.conf:10: Unknown modifiers" → "/usr/lib/tmpfiles.d/20-systemd-shell-extra.conf:10: unknown modifiers". In normal writing, the part of the sentence that follows a colon is not capitalized, so I'd drop it here too.

But you are right that the surrounding messages use uppercase. It doesn't make sense to change just one message, so I dropped that change now.